k8s/configMap-sjc.yaml

```diff
@@ -33,3 +33,6 @@ data:
       - Id: services-all
         FrontendRegexp: ^[^\.]+\.k8s\.wikia\.net
         SamplingRate: 1.0
+      - Id: helios
```

Comment:
The questions here is - do we want helios metrics under both rules (`services-all` and `helios`), or should we make some changes to avoid this overlap?